pset6/mario.py

```diff
@@ -0,0 +1,14 @@
+#http://docs.cs50.net/problems/mario/more/mario.html
+def build_pyramid(h):
+    hash = '#'
+    for i in range(1,h + 1):
```

Comment:
This is pretty hard to read. Can you write it in a way that makes it more obvious what you're doing?
```
spaces = ' ' * (number of spaces)
blocks = block * (number of blocks)
print spaces + blocks + ' ' + blocks + spaces
```